pyproject.toml

```diff
@@ -10,25 +10,11 @@ packages = [
 
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
 astro-kittens = "^1.4.6"
-numpy = [
-    { version = ">=2.0.0", python = ">=3.12"},
-    { version = "<2.0.0", python = ">=3.8, <3.12"}
-]
-scipy = [
-    { version = ">=1.13.0", python = ">=3.12"},
-    { version = "<1.13.0", python = ">=3.8, <3.12"}
-]
-astlib = [
-    { version = ">=0.12.0", python = ">=3.12"},
-    { version = "<=0.11.10", python = ">=3.8, <3.12"},
-    { version = "<=0.11.8", python = "<3.8"}
-]
-astropy = [
-    { version = ">=6.1.0", python = ">=3.12"},
-    { version = "<6.1.0", python = ">=3.8, <3.12"}
-]
+numpy = ">=2"
```

Comment:
Any reason to be this restrictive on the numpy version? Aside from the WCS changes I don't see an immediate reason to force numpy 2.0

Reply:
I've taken that back. Please review latest version.